### PR TITLE
Encapsulation , Inheritance

### PR DESCRIPTION
**Pull request by VurfanHussainy**

### CODE REVIEW

File: Vurfan/Encapsulation.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+class Student {
+  constructor(name, marks) {
+    this.name = name;
```
Comment (meesaw) [inline Vurfan/Encapsulation.js:3]: where is the declaration for field name and marks?

File: Vurfan/Inheritance.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    this.emp_name = name;
+    this.emp_age = age;
+    this.emp_gender = gender;
+    this.epm_designation = designation;
+    this.emp_location = location;
+    this.emp_skills = skills;
+  }
+}
+
+class Comapany extends Employee {
```
Comment (meesaw) [inline Vurfan/Inheritance.js]: Is-A test is failing here.

Company is not an employee. Company can have employees though

File: Vurfan/Encapsulation.js (hunk 1 above)
Comment (VurfanHussainy) [reply Vurfan/Encapsulation.js:3]: Showing error when field name and marks are declared inside class

File: Vurfan/Inheritance.js (hunk 2 above)
Comment (VurfanHussainy) [reply Vurfan/Inheritance.js]: Have modified the code and written in correct pattern .

**Pull request merged**